### ISSUE TITLE
When will the colab evaluation server be released?

### ISSUE THREAD

**Issue by Chan-Sun** [eccv22-ood-workshop/ROBIN-dataset#8]
Hi! When will the colab evaluation server be released? thx

**Comment by DTennant** [comment]
Hi,
Thanks again for your interest in our workshop, and we apologise for the late update.
I have update the README in this repo and update a link to the full test set and labels in the README, the codalab submission server will also be online within this week.

Here are some changes to the competition:

The Phase-1 of the competition will not be a code submission challenge, we have released all the test data and labels in this repo. And Phase-1 will last longer than original planed, we will ask each team to provide a description of their developing environment at the end of Phase-1, Phase-2 will still be code submission challenge.
We will be using Top-1 accuracy for image-classification, mAP@50 for object detection, and Acc@pi/6 for pose estimation as the metric, the IID test performance will also be considered as per request of the sponsor, we will penalize submissions that are significantly different in IID performance with our baseline.
The only limitation now is that the model should only be trained on the given training set and/or the ImageNet-1k dataset, no additional dataset is allowed. You can use any ensemble, data augmentation, or test-time training techniques.
Thanks again for your patience.

Best regards

**Comment by Chan-Sun** [comment]
Thanks for your work!  Quite look forward to this competition!

I notice there is a folder named iid_test besides train folder, so does that mean these iid data are the test data for Phase-1 and ood will appear in Phase-2?

**Comment by DTennant** [comment]
no there is folder named nuisance that is the ood test data. We will be using the iid test data to test the iid performance which will be used to determine whether a submission is valid or not, details will be released after we have confirmed this with our sponsor.

**Comment by Chan-Sun** [comment]
that's clear, thanks.

**Comment by luan-g** [comment]
@DTennant 
Hi,
It seems that the new dataset is different with the previous one. And the numbers of images in the new dataset is different with the numbers in the paper. What’s changed？

**Comment by DTennant** [comment]
Hi,
The codalab evaluation server is released, you can checkout the README to find the link to the server
